src/subscription_manager/plugins.py

```diff
@@ -438,6 +438,26 @@ def __init__(self, clazz, filter_string, installed_products):
         self.installed_products = installed_products
 
 
+class PreUnRegisterConduit(BaseConduit):
```

Comment:
Haha.  I'm not sure if we're going to remove it, but I think we aren't going to put any more effort into expanding it.